frontend/src/metabase/common/components/SaveQuestionForm/context.tsx

```diff
@@ -220,3 +224,34 @@ export const useSaveQuestionContext = () => {
   }
   return context;
 };
+
+/**
+ * Patches form values when `nextValues` change asynchronously (e.g., after API calls).
+ * Only applies updates to fields whose values differ from the initial ones,
+ * avoiding overwriting user input. This is intentional, since patching unchanged
```

Comment:
should it be "changed fields" here? Or maybe rephrase it somehow.
I understand the logic behind the component, but the comment brings more confusion that explanation.